doc/docs_es.md

```diff
@@ -0,0 +1,310 @@
+a constantes y tipos (structs, enums, type, etc).
+
+Nota: `pub` solo se puede usar desde un módulo con nombre.
+Para obtener información sobre cómo crear un módulo, puedes consultar [Módulos](#módulos).
+
+## Variables
+
+```v
+name := 'Bob'
+age := 20
+large_number := i64(9999999999)
+println(name)
+println(age)
+println(large_number)
+```
+
+Las variables se declaran e inicializan con `:=`. Esta es la única
+forma de declarar variables en V. Esto significa que las variables siempre tienen un valor
+inicial.
+
+El tipo de variable se infiere del valor del lado derecho.
+Para elegir un tipo diferente, puedes usar la conversión de tipos:
+la expresión `T(v)` convierte el valor `v` al
+tipo `T`.
+
+A diferencia de la mayoría de los otros lenguajes, V solo permite definir variables en funciones.
+No se permiten variables globales (a nivel de módulo). No hay un estado global en V
+(ver [Funciones puras por defecto](#funciones-puras-por-defecto) para más detalles).
+
+Para lograr coherencia en diferentes bases de código, todos los nombres de funciones y variables
+deben usar el estilo `snake_case`, a diferencia de los nombres de tipo, que deben usar` PascalCase`.
+
+### Variables mutables
+
+```v
+mut age := 20
+println(age)
+age = 21
+println(age)
+```
+
+Para cambiar el valor de la variable usa `=`. En V, las variables son
+inmutables por defecto.
+Para poder cambiar el valor de la variable, debes declararlo con `mut`.
+
+Intenta compilar el programa anterior después de eliminar el `mut` de la primera línea.
+
+### Inicialización vs asignamiento
+
+Ten en cuenta la diferencia (importante) entre `:=` y `=`.
+`:=` se usa para declarar e inicializar, `=` se usa para asignar.
+
+```v failcompile
+fn main() {
+    age = 21
+}
+```
+
+Este código no se compilará porque la variable `edad` no está declarada.
+Todas las variables deben declararse en V.
+
+```v
+fn main() {
+	age := 21
+}
+```
+
+Los valores de múltiples variables se pueden cambiar en una línea.
+De esta forma, sus valores se pueden intercambiar sin una variable intermedia.
+
+```v
+mut a := 0
+mut b := 1
+println('$a, $b') // 0, 1
+a, b = b, a
+println('$a, $b') // 1, 0
+```
+
+### Errores de declaración
+
+En el modo de desarrollo, el compilador te advertirá que no has utilizado la variable
+(obtendrás una advertencia: "unused variable", variable no utilizada).
+En modo de producción (habilitado al pasar la opción `-prod` a v – `v -prod foo.v`)
+no se compilará en absoluto (como en Go).
+
+```v failcompile
+fn main() {
+    a := 10
+    if true {
+        a := 20 // error: redefinition of `a`
+    }
+    // warning: unused variable `a`
+}
+```
+
+A diferencia de la mayoría de los lenguajes, no se permite el shadowing (ocultamiento) de variables. Declarar
+una variable con un nombre que ya se utiliza en un ámbito pariente provocará un error de compilación.
+
+Sin embargo, puede maquillar módulos importados, ya que es muy útil en algunas situaciones:
```

Comment:
@spaceface777 What do you think about this translation?

Reply:
`maquillar` sounds a little strange, but I can't think of anything much better.
`ocultar` is good, but you already use it above, and I'm sure there's another good synonym, but I can't think of any right now.

Reply:
I've never read software documentation in spanish, so I can't take ideas from there :p

Reply:
The thing is a little difficult. Something similar happened with a C++ book translated into Spanish.